tests/common.py

```diff
@@ -502,6 +503,7 @@ def get_fixture_path(filename: str, integration: str | None = None) -> pathlib.P
     )
 
 
+@lru_cache
```

Comment:
I think we can `functools.cache` now that we have a new enough python requirement.

Reply:
The use of `functools.cache` will never evict, meaning it just keeps everything in memory until the session is done. `lru_cache` will keep the max 128 (default) items around.